trino-s3-proxy/src/test/java/io/trino/s3/proxy/server/testing/harness/TrinoS3ProxyTestCommonModules.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.s3.proxy.server.testing.harness;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Binder;
+import com.google.inject.Key;
+import com.google.inject.Module;
+import com.google.inject.TypeLiteral;
+import com.google.inject.multibindings.OptionalBinder;
+import io.trino.s3.proxy.server.remote.RemoteS3Facade;
+import io.trino.s3.proxy.server.testing.ContainerS3Facade;
+import io.trino.s3.proxy.server.testing.ManagedS3MockContainer;
+import io.trino.s3.proxy.server.testing.TestingConstants.ForTesting;
+
+import java.util.List;
+
+public final class TrinoS3ProxyTestCommonModules
+{
+    public static final class WithConfiguredBuckets
+            implements Module
+    {
+        public static final List<String> CONFIGURED_BUCKETS = ImmutableList.of("one", "two", "three");
```

Comment:
As this is bound with Guice now, make it private and consume it via injection in the tests that need it.